runtime/src/iree-amd-aie/aie_runtime/CMakeLists.txt

```diff
@@ -503,6 +505,7 @@ iree_cc_library(
     ${_aie_rt_binary_include_dir}
     # For <elf.h>
     ${_bootgen_source_dir}
+    ${_amd_aie_attrs_gen_binary_dir}
```

Comment:
Could we add it as a dependency instead? With this I am seeing duplicated definitions in files where I need both the `AMDAIEDialect` and `aie_runtime`.

Reply:
You can't but that also shouldn't be happening - what CMake args are you using?

Reply:
Oh sorry in files where you include both. Yes that's true but there's a simple fix - the enum inc file just needs its own header with a header guard (inc files don't have header guards). Will do shortly.

Reply:
See for example: https://github.com/nod-ai/iree-amd-aie/compare/jornt_cpp_pipeline?expand=1#diff-84a9775533b1da06639c4448d69c3c87b7b4517831e3daf6064a3618256978d0R493

Reply:
err duh ok that's better